### PR TITLE
Animate creeps

### PR DESCRIPTION
**Pull request by KendallSpackman**
This adds spritesheet animation to the creeps.
The rotation functionality is still broken. (It actually seems worse. Maybe I just didn't notice it as much before.) I have an idea, so I'll see if I can get it done tonight.
@austinstoker

### CODE REVIEW

**Comment by austinstoker** [comment]
what are pdn files?  I missed the days on sprites so maybe that's why I don't know...

**Comment by KendallSpackman** [comment]
They're paint.net files. He recommended that to join the images into one file. We probably don't need those files in the repo though. Let me take them out quick.

File: scripts/Gameplay/Components.js [hunk 1]
```diff
@@ -49,8 +49,8 @@ DTD.components = (function(graphics) {
 	var Constants = {
     get GridHeight() { return 20; },
     get GridWidth() { return 20; },
-    get CreepHeight() { return 20; },
-    get CreepWidth() { return 20; },
+    get CreepHeight() { return 40; },
+    get CreepWidth() { return 40; },
```
Comment (KendallSpackman) [inline scripts/Gameplay/Components.js]: I like the creeps only being one tile/cell big. But you couldn't even see the animation at that size. So I made them bigger, at least for the deliverable.

Comment (austinstoker) [inline scripts/Gameplay/Components.js]: Yeah they were too small before maybe we want to make creeps 30, grid 30 and towers 60, or just make everything 40... anyway that should be a pretty simple change if we decide to make it later.

**Comment by austinstoker** [comment]
I see what you mean about the creeps rotation.  That is different than it was before the animation.  I also notice more lag when placing a tower, maybe because it has to do a little more animation work in addition to it's recalculating complete shortest path every time.  I have an idea on how to make it do less work on the path finding.

**Pull request merged**